hq/app/schedule/IamNotifier.scala

```diff
@@ -20,15 +20,20 @@ object IamNotifier extends Logging {
 
   def send(
     notification: Notification,
-    topicArn: String,
+    topicArn: Option[String],
     snsClient: AmazonSNSAsync)(implicit executionContext: ExecutionContext): Unit = {
-    val response = Anghammarad.notify(notification, topicArn, snsClient)
-    try {
-      val id = Await.result(response, 5.seconds)
-      logger.info(s"Sent notification to ${notification.target}: $id")
-    } catch {
-      case NonFatal(err) =>
-        logger.error("Failed to send notification", err)
+    topicArn match {
+      case Some(arn) =>
+        val response = Anghammarad.notify(notification, arn, snsClient)
+        try {
+          val id = Await.result(response, 5.seconds)
```

Comment:
rather than await result can you do `response.foreach(id => logger.info(s"Sent notification to ${notification.target}: $id"))`